src/Components/StyledPostCard.js

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components';
```

Comment:
Here you mixed up the styled components components :) please use one file per styled component. Your Coaches will talk about it soon, since this is a common issue in your class :)